diff --git a/node_modules/@mgcrea/react-native-dnd/dist/DndProvider.js b/node_modules/@mgcrea/react-native-dnd/dist/DndProvider.js
index 2bbf21f..169379f 100644
--- a/node_modules/@mgcrea/react-native-dnd/dist/DndProvider.js
+++ b/node_modules/@mgcrea/react-native-dnd/dist/DndProvider.js
@@ -109,6 +109,7 @@ export const DndProvider = forwardRef(function DndProvider({ children, springCon
             }, delay);
         };
         const panGesture = Gesture.Pan()
+            .activateAfterLongPress(activationDelay > 0 ? activationDelay : 200)
             .onBegin((event) => {
             const { state, x, y } = event;
             debug && console.log("begin", { state, x, y });
